docker-java-api/src/main/java/com/github/dockerjava/api/command/LoadImageAsyncCmd.java

```diff
@@ -0,0 +1,32 @@
+package com.github.dockerjava.api.command;
+
+import com.github.dockerjava.api.model.LoadImageAsyncResponseItem;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+import java.io.InputStream;
+
+/**
+ * load image async and return image id or tag
+ * @author luwt
```

Comment:
Could you please remove `@author` and `@date` tags here and in other places, as we're not using them?

Reply:
ok, i will remove it